<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuFind_by">
     <property name="title">
      <string>Find By...</string>
     </property>
     <addaction name="actionRegistration_numder"/>
     <addaction name="actionAuthor"/>
     <addaction name="actionTitle_and_Year_of_publication"/>
    </widget>
    <addaction name="actionAddBook"/>
    <addaction name="actionDeleteBook"/>
    <addaction name="actionChange"/>
    <addaction name="actionShow_All"/>
    <addaction name="menuFind_by"/>
    <addaction name="actionSort_By_Author"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAddBook">
   <property name="text">
    <string>Add Book</string>
   </property>
  </action>
  <action name="actionDeleteBook">
   <property name="text">
    <string>Delete Book</string>
   </property>
  </action>
  <action name="actionShow_All">
   <property name="text">
    <string>Show All</string>
   </property>
  </action>
  <action name="actionRegistration_numder">
   <property name="text">
    <string>Registrational Number</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionTitle_and_Year_of_publication">
   <property name="text">
    <string>Title and Year of Publication</string>
   </property>
  </action>
  <action name="actionChange">
   <property name="text">
    <string>Change</string>
   </property>
  </action>
  <action name="actionSort_By_Author">
   <property name="text">
    <string>Sort By Author</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddBook</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionAddBook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAuthor</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionFindAuthor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDeleteBook</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionDeleteBook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRegistration_numder</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionFindRegistNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_All</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionShowAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTitle_and_Year_of_publication</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionFindTitleYear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChange</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionChangeBook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSort_By_Author</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionSort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionOpen()</slot>
  <slot>actionAddBook()</slot>
  <slot>actionDeleteBook()</slot>
  <slot>actionShowAll()</slot>
  <slot>actionFindRegistNum()</slot>
  <slot>actionFindAuthor()</slot>
  <slot>actionFindTitleYear()</slot>
  <slot>actionChangeBook()</slot>
  <slot>actionSort()</slot>
 </slots>
</ui>
